### DISCUSSION TITLE
Notifications issue

### DISCUSSION THREAD

**Discussion by jasnicaa** [GetStream/stream-chat-flutter#792] [Q&A] [answered]
Why is Firebase showing notifications in one-to-one chat on get stream like this and how to fix it?  I can see that it is a chat id actually channel id but I don't want to show that in a notification, another option is to put the channel's name but then it overrides in channels list view default one, and if I have let's say 5 private chat's they would all be the same.. 

![WhatsApp Image 2021-11-26 at 10 30 26 AM](https://user-images.githubusercontent.com/25682164/143558710-63e97e07-37d9-4e04-84ee-3e6ab5918f79.jpeg)

**Comment by imtoori** [comment, marked as answer]
did you try changing the notification template in the dashboard? I think that's the solution

**Reply by jasnicaa** [reply]
yeah I did, but somehow it was overridden... let me see what is about :/
